doc/adr/0008-jetstream-kv.md

```diff
@@ -0,0 +1,326 @@
+# 8. JetStream based Key-Value Stores
+
+|Metadata|Value|
+|--------|-----|
+|Date    |2021-06-30|
+|Author  |@ripienaar|
+|Status  |Partially Implemented|
+|Tags    |jetstream, client, kv|
```

Comment:
@scottf here is an example of how I suggest we change the top-of-adr, we can then parse these tags and provide a `client` only list of associated ADRs etc, giving us the ability to have many views into the growing set of ADRs

We can list unapproved ones, approved ones, ones by impact etc.

Reply:
Here's an example of the Python PEP process.  They have 1 PEP list, PEP-1, PEP-2 etc much like the ADR process.

What they then have is views/indexes over these for like "Accepted PEPs" and so forth.

https://www.python.org/dev/peps/

The intention with the tags here is to have generated indexes for all ADRs that might need client implementations, or ones thats just kv related, or jetstream related - or even an intersection of the lot so one ADR will show up in 3 indexes in this case.

I don't think we can solve this with a file naming convention, so interested in views from the reviewers here, if we think this is a useful thing I can update the other ADRs at the same time to have this format